src/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.jsx

```diff
@@ -2,14 +2,18 @@ import Box from '@mui/material/Box'
 import Avatar from '@mui/material/Avatar'
 
 import AppRating from '~/components/app-rating/AppRating'
+import AppRatingMobile from '~/components/app-rating-mobile/AppRatingMobile'
 
 import { styles } from '~/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.styles'
 
-const OfferAvatarAndRating = ({ imgSrc, rating }) => {
+const OfferAvatarAndRating = ({ imgSrc, rating, totalReviews }) => {
   return (
     <Box sx={styles.container}>
       <Avatar src={imgSrc} sx={styles.avatar} />
       <AppRating readOnly showNumber sx={styles.rating} value={rating} />
+      <Box sx={styles.reviewsCountContent}>
```

Comment:
this box is no needed too

Reply:
i tried without this box, but it doesnt work, and i didnt understand how to fix it without changing AppRatingMobile.styles.ts

Reply:
<Typography variant={'caption'}>
        {t('tutorProfilePage.reviews.reviewsCount', {
          count: totalReviews})}
      </Typography>
just put it here and style as needed, also put in prop totalReviews reviews according to userRole, not my role ad in AppRatingMobile, that file has mistake